meshtastic/node_runtime/seturl/rollback.py

```
@@ -0,0 +1,311 @@
+    def _rollback_add_only(
+        self,
+        *,
+        admin_context: _SetUrlAdminContext,
+        plan: _SetUrlAddOnlyPlan,
+        state: _SetUrlAddOnlyExecutionState,
+    ) -> None:
+        """Run best-effort rollback for addOnly transactions."""
+        logger.warning(
+            "Failed while applying addOnly channel updates; attempting rollback "
+            "for written channels and LoRa config.",
+            exc_info=True,
```

Ruff (0.15.7)

[warning] `exc_info=` outside exception handlers

Remove `exc_info=`

(LOG014)